pkg/cmd/install.go

```diff
@@ -142,6 +142,7 @@ func newCmdInstall(rootCmdOptions *RootCmdOptions) (*cobra.Command, *installCmdO
 	cmd.Flags().StringArray("node-selector", nil, "Add a NodeSelector to the operator Pod")
 	cmd.Flags().StringArray("operator-resources", nil, "Define the resources requests and limits assigned to the operator Pod as <requestType.requestResource=value> (i.e., limits.memory=256Mi)")
 	cmd.Flags().StringArray("operator-env-vars", nil, "Add an environment variable to set in the operator Pod(s), as <name=value>")
+	cmd.Flags().StringP("log-level", "v", "info", "The level of operator logging (default - info): info or 0, debug or 1")
```

Comment:
In `kamel run`, `-v` is already used for volume. For consistency throughout cli isn't it good idea to avoid overloading of short flags and use a different one?  For example, what about using `-V` instead?

Reply:
Also, at #3193 @johnpoth is introducing `--verbose` flag for `kamel run`. Is it something we can provide consistency for those similar functions?

Reply:
I'll change the letter. Didn't notice the overloading. Thanks @tadayosi!